### DISCUSSION TITLE
I downgraded Copilot trial to monthly but still got yearly in Add-ons

### DISCUSSION THREAD

**Discussion by Moustafa3attia** [community/community#58232] [Copilot Conversations] [answered]
### Bug

Hi Community,

I was playing around, subscribed to copilot yearly plan, got my trial started just yesterday.
Today, I decided to go with monthly one, clicked downgrade, and now I have "**Your next payment $10.00 Due by July 16, 2023**" in the Billing section but still got "**You'll be billed $100.00/year after the trial ends on Jul 15, 2023. Cancel anytime.**" in the Add-ons section.

Any hints are appreciated.
Regards.

**Comment by Akash1134** [comment, marked as answer]
Hi @Moustafa3attia  The best place to go for answers to account-specific or billing questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by Moustafa3attia** [reply]
Thanks Akash for the forward, will do so.
